server_installation/topics/operator/installation.adoc

```diff
@@ -0,0 +1,145 @@
+
+[[_installing-operator]]
+=== Installing the {project_operator} on a cluster
+
+To install the {project_operator}, you can use:
+
+* <<_install_by_olm,The Operator Lifecycle Manager (OLM)>> 
+* <<_install_by_command,Command line installation>> 
+
+[[_install_by_olm]]
+==== Installing using Operator Lifecycle Manager
+
+ifeval::[{project_community}==true]
+You can install the Operator on a OpenShift or Kubernetes cluster
+
+===== Installation on an OpenShift cluster
+endif::[]
+
+.Prerequisites
+
+You have cluster administrator permissions or an equivalent level of permissions granted by an administrator.
```

Comment:
cluster-admin permission

Reply:
@andymunro I do not know is you observed this comment. So I am just assigning it to you. Thank you for your help. :-)

Reply:
I found in the OCP docs that they specify "cluster-admin" permission/role in the prerequisites.